src/blob_info.rs

```diff
@@ -12,37 +12,12 @@ use super::{
     },
 };
 
-#[derive(Debug)]
-pub enum ConversionError {
-    NotPresentError,
-}
-
-impl fmt::Display for ConversionError {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        match self {
-            ConversionError::NotPresentError => write!(f, "Failed to convert BlobInfo"),
-        }
-    }
-}
-
 #[derive(Debug, PartialEq, Clone)]
 pub struct G1Commitment {
     pub x: Vec<u8>,
     pub y: Vec<u8>,
 }
 
-impl G1Commitment {
-    pub fn to_bytes(&self) -> Vec<u8> {
-        let mut bytes = vec![];
-        bytes.extend(&self.x.len().to_be_bytes());
-        bytes.extend(&self.x);
-        bytes.extend(&self.y.len().to_be_bytes());
-        bytes.extend(&self.y);
-
-        bytes
-    }
```

Comment:
why are you deleting all these to_bytes? Were they used for something before?

Reply:
they were not needed.